frontend/interfaces/mdm.ts

```diff
@@ -14,7 +14,7 @@ export interface IMdmAppleBm {
 }
 
 export interface IMdmEnrollmentCardData {
-  status: "Enrolled (manual)" | "Enrolled (automatic)" | "Unenrolled";
+  status: "On (manual)" | "On (automatic)" | "Off";
```

Comment:
double checking: did you omit `"Pending"` intentionally? (meant to be added by the person that implements the UI for this?)